README.md

```diff
@@ -1,5 +1,53 @@
-# curriculum-cr-training-contact-form
 
-This repo host an initial code for the tasks performed during the Code Reviewers training.
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-Application is based on this [Figma design](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471).
+
+# Portofolio
+
+## Description
+
+> This is my  portofolio which discribe my qualification. And here you found my recent works.  
+
+## Website Mockup 📱 💻 🖥️
+![all-devices-black](https://user-images.githubusercontent.com/95297251/172169864-395711b0-f62a-4aa6-be75-0040027ddbf5.png)
+
+
+## Built With
+
+- HTML
+- CSS
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+1. Download the `Zip` or `(git clone)[https://]`
+2. `cd Hello-Microverse`
+3. Then open in your browser by using `live server`
+
+## Authors
+
+👤 **Magana Assia**
+
+- GitHub: [@MaganaAsiati ](https://github.com/MaganaAsiati)
+- Twitter: [@asia_magana](https://twitter.com/asia_magana)
+- LinkedIn: [Asia Magana](https://www.linkedin.com/in/asia-magana-60b451200/)
+
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Thank you Microverse
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] [OPTIONAL] On your readme, you have the section "license" according to the [Microverse readme template](https://github.com/microverseinc/readme-template).  Kindly, make sure that The link featured in that section should point to a file at the root of your project (the license file).   Just copy it from the mentioned template:

![image](https://user-images.githubusercontent.com/57421823/168004669-c3026da8-8102-4a87-bb49-f7a5a9a6abbf.png)

That way, you will have a professional repo, and your project will be well documented 💪